RadixWallet/Features/AssetsFeature/Components/HelperViews/GeneralHelperViews.swift

```diff
@@ -6,21 +6,21 @@ struct KeyValueView<Content: View>: View {
 	let key: String
 	let content: Content
 
-	init(resourceAddress: ResourceAddress) where Content == AddressView {
+	init(resourceAddress: ResourceAddress, imageColor: Color?) where Content == AddressView {
```

Comment:
seems we mostly use `.app.gray2` ? should we maybe default to that?

Reply:
good point 👍